spring-xd-integration-test/src/main/java/org/springframework/xd/integration/fixtures/Sources.java

```diff
@@ -343,4 +345,14 @@ public TcpClientSource tcpClientSource() {
 		return new TcpClientSource(xdEnvironment.getTcpClientHost(), xdEnvironment.getTcpClientPort());
 	}
 
+	/**
+	 * Creates a ZkClient connected to the Zookeeper that the kafka is associated.
+	 * @return zkClient instance that is connected to the kafka's zookeeper
+	 */
+	public ZkClient getKafkaZkClient(){
```

Comment:
This doesn't correspond to a real source, it should be moved somewhere else... thinking of where..